setup.py

```diff
@@ -29,8 +27,6 @@
           'Topic :: Multimedia :: Graphics :: 3D Rendering',
           'Topic :: Scientific/Engineering :: Visualization',
     ],
-    install_requires=['jupyter', 'vpython'],
-    #cmdclass=cmdclass('vpnotebook/data'),
-    #package_data={'vpython': ['data/*'], 'vpnotebook': ['data/kernel.json']},
+    install_requires=['jupyter', 'vpnotebook'],
```

Comment:
`numpy` is also currently a dependency, so I'd list it here. When that dependency is removed we can always update `install_requires`.